tests/TestHarness/Cluster.py

```diff
@@ -486,6 +481,7 @@ def connectGroup(group, producerNodes, bridgeNodes) :
             return False
 
         startedNodes=totalNodes-unstartedNodes
+        self.nodes=list(range(startedNodes)) # placeholder for cleanup purposes only
```

Comment:
Is this a TODO for this PR or for later down the line?

Reply:
That's a merge error.  The line was removed and the merge put it back.  It's nonsense that doesn't work.